### ISSUE TITLE
As a team player I would like to know the stats for my team

### ISSUE THREAD

**Issue by matt-cassidy** [alexinman/elo-slack-app#18]
I would like to see this be able to work
```
/elo stats <optional game> @matthew.cassidy and @alex.inman
```
And the stats would be filtered down to just that team
```
@matthew.cassidy and @alex.inman
Rank
3. @matthew.cassidy and @aaron.broad (1024)
4. @josh.northrup and @ming.wong (1023)
4. @matthew.cassidy and @alex.inman (1023)
5. @matthew.cassidy and @jamie.mcfarlane (1014)
Wins
2
Losses
0
Recent Games
• Jun 28 at 4:43 PM: Beat @mike.stewart and @brion.macdonald
• Jun 14 at 6:05 PM: Beat @mike.stewart and @brion.macdonald
```

**Comment by alexinman** [comment]
I think this will be easier to tackle once we've implemented the CommandParser class mentioned in #16

**Comment by alexinman** [comment]
Done by #34
